portfolio/src/main/webapp/aboutME.html

```diff
@@ -55,24 +54,23 @@ <h1>Random Ade</h1>
             role <br>in making it better through education and he worked hard to achieve that. 
             Consequently, he was<br> able to secure a scholarship to get his bachelor’s degree 
             in the United States and fulfill <br> his dream of getting an education.
-          </p>
-          <br>
-          <br>
-          <p style = "font-size:20px;color: grey" align = "center">
+        </p>
+        <br>
+        <br>
+        <p>
             Even though he lived most of his life in Nigeria where blacks are the 
             majority, <br>there was still diversity as there are different people from 
             different cultures and his life <br> experiences opened him up to accepting and 
             appreciating those who are culturally diverse from him. while <br>his ability to 
             form relationships with diverse individuals has made him more appreciative of 
             other <br>races, he understands that this does not mean that he knows what life is 
             like as a<br> minority. He has lived his whole life comfortably as part of the majority 
-            group in Nigeria<br>. It wasn’t until he got to college that he really started interacting 
-            with people of different<br> races and cultures. Despite the odds, here he is, and here 
-            he hopes to stay for a<br> period of time, not only culturally enriched and happy to broaden 
-            his horizons but also studying<br> hard to get a college degree to create opportunities and 
-            be an inspiration to young kids<br> like him who are bold enough to want more from life. 
-          </p>
+            group in Nigeria<br>. 
+        </p>
       </div>
     </div>
+    <div class="footer">
+      <p>Ade </p>
+    </div>
   </body>
-</html>
```

Comment:
Please fix "no newline at end of file", some interesting reading: https://thoughtbot.com/blog/no-newline-at-end-of-file

Reply:
i will have it fixed! Thanks alot shenqiu! i don't know why for some reason it does that by itself. i will make sure to always fix that moving forward!